Assets/Scripts/Runtime/Scriptable Objects/AutomaticSourceSpawning.cs

```diff
@@ -55,7 +57,12 @@ private Vector3Int GetSpawnPosition()
             var zCenter = (maxZ + minZ) / 2;
             var offset = new Vector3(xCenter, yCenter, zCenter);
 
-            var radius = 5f;
+            var averageDistToRelativeCenter = (
+                (Math.Max(maxX, Math.Abs(minX)) - xCenter)+
+                (Math.Max(maxY, Math.Abs(minY)) - yCenter)+
+                (Math.Max(maxZ, Math.Abs(minZ)) - zCenter)) /3;
+
+            var radius = averageDistToRelativeCenter + 5f;
```

Comment:
Can you put 5f in a named variable so we don't have magic numbers?

Reply:
yeah that makes sense